platform/util/platform-util/src/main/java/org/codice/ddf/platform/util/RandomNumberGenerator.java

```diff
@@ -22,11 +22,11 @@
 
 public class RandomNumberGenerator {
```

Comment:
I don't believe that this qualifies as a public API. This is a utility class that can be embedded. Developers are free to embed the older version if they choose.